data/ideas.json

```
@@ -9,29 +9,29 @@
         "title": "Snake",
         "description": "If you had a cellphone in the early 2000s, you’ve probably played Snake. It’s a simple game but you can always try to make things more complex! First, start by just trying to make the game.",
         "level": "intermediate",
-        "tags": "snake,game"
+        "tags": "snake, game"
     },
     {
         "title": "Tetris",
         "description": "Create a Tetris game! Sure, it has been done a million times, but it's really interesting to try to make it!",
         "level": "intermediate",
-        "tags": "tetris,game,classic"
+        "tags": "tetris,game, classic"
```

Add space before `game`
```suggestion
        "tags": "tetris, game, classic"
```